src/entities/activity/lib/services/AnswersUploadService.ts

```diff
@@ -144,12 +167,22 @@ class AnswersUploadService implements IAnswersUploadService {
       return remoteUrl!;
     } catch (error) {
       throw new Error(
-        `[UploadAnswersService.uploadFilesForAnswer]: Error occurred while file ${logFileInfo} uploading\n\n` +
+        `[UploadAnswersService.processFileUpload]: Error occurred while file ${logFileInfo} uploading\n\n` +
           error!.toString(),
       );
     }
   }
 
+  private logFilesUploadCheck(
+    uploadChecks: CheckFilesUploadResults,
+    position: '1' | '2',
+  ) {
+    this.logger.log(
+      `[UploadAnswersService.uploadAllMediaFiles] Check if files uploaded #${position}:\n\n` +
```

Comment:
Please rename it to "...Check if **log files** uploaded..."

Reply:
Here we check regular files, not logs